capstone-frontend/src/components/SearchUserModal.jsx

```diff
@@ -74,6 +78,32 @@ export class SearchUserModal extends Component {
 
   componentDidMount() {
     this.fetchCollaborators();
+    this.fetchFriends();
+
+    // TODO: Update ClubPage based on user input
+
+    if (this.props.club && this.props.type === 'clubs') {
+      if (this.props.club.memberIDs) {
+        if (this.props.club.memberIDs.length > 0) {
```

Comment:
It looks like you already have a check for members > 0 in the fetchMembers function to see if there are no members. Do you need this check here then?

Reply:
Yes. Because of ```this.props.club``` can return an empty object ```{}```, thus ```memberIDs``` will be undefined. Consequently, if you try to get the ```.length``` of an undefined variable, React will throw an error and stop the web app.  This is why I have to do multiple checks. 

Note: An empty object ```{}``` is consider to be truthy. ```undefined``` is falsy.

Reply:
This might be another point in favor of upgrading `memberIDs` to the top-level state in this component